src/backend/src/lib.rs

```diff
@@ -0,0 +1,94 @@
+use ic_cdk::export::candid::CandidType;
+use ic_cdk_macros::*;
+use ic_evm_sign;
+use ic_evm_sign::state::ChainData;
+
+#[derive(Debug, CandidType)]
+struct CreateResponse {
```

Comment:
Rename `CreateResponse`, `SignatureInfo`, `DeployEVMContractResponse` and `CallerResponse` with same names as in the library